files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -232,56 +232,6 @@
     - 'delphix'
     - 'root'
 
-#
-# The 'ClientAliveInterval' setting determines the amount of time
-# (in seconds) the sshd server will wait to receive data from the
-# client before sending a request for response.
-#
-- set_fact:
-    ssh_client_alive_interval: "300"
-    ssh_client_alive_count_max: "3"
-
-#
-# With that said (see comment above), the Azure marketplace does not
-# allow a value greater than 3 minutes. So, when running on Azure, we
-# use 3 minutes.
-#
-- set_fact:
-    ssh_client_alive_interval: "180"
-    ssh_client_alive_count_max: "0"
-  when:
-    - platform == "azure"
-
-- lineinfile:
```

Comment:
so.. the issue with removing ansible logic like this, is it doesn't undo any configuration that might have been already applied..

do we need to undo anything? is it safe to just stop applying these changes, and keep the `/etc/ssh/sshd_config` file at w/e state it's in? considering it might have been modified by this logic in a prior release?

Reply:
I thought about that, and I concluded that we can abandon the changes we made to the `/etc/ssh/sshd_config` file given that the override will always take precedence over those old changes (we're creating an override that sets the exact same values that used to be set in `/etc/ssh/sshd_config`). We could implement logic to reset the file to its as-shipped contents, too, but I didn't think it was necessary.